helm/kanister-operator/values.yaml

```diff
@@ -37,6 +37,10 @@ controller:
     # false : kanister-prometheus framework has been disabled
     # true: kanister-prometheus framework has been enabled
     enabled: false
+  # labels to set on the pods created by kanister functions
+  kanisterPodCustomLabels : ""
```

Comment:
What is the value format? Is it a map of key-value or comma-separated pair?
We should add a comment/example about that.

Reply:
It's documented separately (part of this PR itself) as part of the docs.

Reply:
https://github.com/kanisterio/kanister/pull/2296/commits/b20b725953509afe4417986faa47b2f6d1744b8f

Reply:
I think we should also add a comment here as well.